OpenEdXMobile/res/layout/activity_launch.xml

```diff
@@ -6,9 +6,16 @@
         android:id="@+id/rl_root"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        tools:background="@drawable/launch_background"
         android:splitMotionEvents="false">
 
+        <ImageView
```

Comment:
@adamsythe There is no need to add Image view or any other change in the code, It can be achieve by placing the correct sizes drawables for hdpi, mdpi, xdpi, xxdpi and xxxhdpi.

Reply:
@adamsythe Just a reminder,  we are planning to close this PR, as the required results can simply be achieved by placing the different densities images into relevant folders.
Please respond if you think otherwise.